extensions/odoo_theme/translator.py

```diff
@@ -10,15 +10,12 @@
 #         └── Odoo Translator
 
 ADMONITION_MAPPING = {
-    # ???: 'alert-success',
-
-    'note': 'alert-note',
-
-    'hint': 'alert-info',
+# The alert classes have been replaced by default BS classes to reduce number of scss lines.
```

Comment:
Don't write changelogs in the code. It won't mean anything to new readers :wink: 
```suggestion
```